### DISCUSSION TITLE
OSPOlogy Follow-up: Measuring the Business Impact of Open Source & OSPOs

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#50] [OSPO Stories]
Discuss with the community topics related to the upcoming OSPOlogy meeting: 

📅 Feb, 2nd [Measuring the Business Impact of Open Source & OSPOs](https://community.linuxfoundation.org/events/details/lfhq-todo-group-presents-measuring-the-business-impact-of-open-source-ospos/)

In this open space, people can:

* Ask questions prior to the meeting starts
* Share notes or takeaways from the conversation
* Make proposals for new initiatives that can help the OSPO Ecosystem
* Add useful links and resources related to the topic
* Connect with mentors involved in OSPOs and/or Open Source Management

**Comment by justinabrahms** [comment]
One thing I'm trying to figure out: How do you quantify "did our reputation for open source influence your desire to apply for this job?"-style questions for improving recruiting.  I have a theory that there are some people who join companies b/c of their position in open source (redhat, google, etc). Wanted to quantify it at eBay, but unsure how others are doing it.

**Reply by amcasari** [reply]
As I review the different parts of this question, I come back to the idea that asking anything of candidates in the interview process will potentially always carry a bias. The company is in a position of power which doesn't fully allow for transparency. 

You may be able to ask folks questions who are already hired during their onboarding process. If you have them complete any onboarding training related to open source, it would be one place to drop in a short survey.

**Comment by LayZeeDK** [comment]
Question: What are good examples of Google collaborating with industry rivals?

**Reply by juliaferraioli** [reply]
I can take a stab at this. Off the top of my head, Kubernetes is a prime example, and to an extent Tensorflow and Go. Those are just big ones, not counting less well-known collaborations.

**Reply by jdorfman** [reply]
Envoy Proxy is another big one (Used by Microsoft, AWS, Dropbox). Anna Berenberg talks about collaborating with the Envoy Community [here](https://podcast.curiefense.io/17) (shameless plug).

**Comment by juliaferraioli** [comment]
Information on OCEAN (Open source Complex Ecosystems And Networks):

* https://github.com/google/project-OCEAN
* https://vermontcomplexsystems.org/partner/OCEAN/

**Comment by LayZeeDK** [comment]
Question: How do you address stakeholders worrying about the financial aspect of collaborating with or providing software used by competitors?

**Comment by juliaferraioli** [comment]
Useful metrics, a framework (Preventing Random Acts of Metrics): https://bit.ly/foss-backstage-pram

**Reply by awright** [reply]
i'm really excited for this presentation. thank you! ;)

**Reply by amcasari** [reply]
We would love feedback or questions, if you have any!

**Comment by anajsana** [comment]
Google open source 2020 metrics: bit.ly/google-opensource-2020-metrics

**Reply by justinabrahms** [reply]
https://bit.ly/google-opensource-2020-metrics just to make it clickable.

**Comment by anajsana** [comment]
@jlprat asks:

Question regarding blog.google/technology/safety-security/making-open-source-software-safer-and-more-... -- One of the key factors is to identify crucial projects, but this might not really reflect which projects are in most need of help. How to overcome this?

**Reply by anajsana** [reply]
@juliaferraioli replies

"that's such a good question, especially as "help" comes in many forms. If a project is well funded and supported, even if it's critical, you might want to prioritize others with your time, talent, and treasure."

**Reply by jlprat** [reply]
Yes I'm a big proponent of "help does does not equal funding". Personally, I'd prefer companies focus on "donating" people's time to focus on projects by contributing in different levels

**Comment by LayZeeDK** [comment]
Question: What are good examples of OSS products with open governance that Google are heavily involved in?

**Comment by jlprat** [comment]
Some more links shared:
https://openssf.org/press-release/2022/02/01/openssf-announces-the-alpha-omega-project-to-improve-software-supply-chain-security-for-10000-oss-projects/
https://opencollective.com/foss-responders

**Reply by will-ww** [reply]
Great, many thanks~
